CondFormats/HcalObjects/interface/HcalL1TriggerObjects.h

```diff
@@ -10,6 +10,9 @@
 #include "CondFormats/HcalObjects/interface/HcalL1TriggerObject.h"
 #include "CondFormats/HcalObjects/interface/HcalCondObjectContainer.h"
 
+#include "FWCore/Utilities/interface/Exception.h"
+
+constexpr std::size_t charArraySize = 128;
```

Comment:
I believe this constant would be better placed as a member of `HcalL1TriggerObjects` to avoid it being in the global namespace.

Reply:
Done. That is better. Thanks.